lib/matplotlib/tri/tricontour.py

```diff
@@ -79,12 +79,22 @@ def _contour_args(self, args, kwargs):
             fn = 'contour'
         tri, args, kwargs = Triangulation.get_from_args_and_kwargs(*args,
                                                                    **kwargs)
-        z = np.asarray(args[0])
+        z = np.ma.asarray(args[0])
         if z.shape != tri.x.shape:
             raise ValueError('z array must have same length as triangulation x'
                              ' and y arrays')
-        self.zmax = z.max()
-        self.zmin = z.min()
+
+        # z values must be finite, only need to check points that are included
+        # in the triangulation.
+        z_check = z[np.unique(tri.get_masked_triangles())]
```

Comment:
I would rather call this `z_used`. It's better to name it after what it is (*used* in the triangulation) than what you want to do with them.

Reply:
@timhoffm `z_check` is a subset of `z` and is only used to check values are valid. The contouring uses the original `z`, as is shown by the end of the function:

    self._contour_level_args(z, args[1:])
    return (tri, z)

Hence `z_check` is the correct name.